.github/workflows/platform.convertToArmTemplate.tests.yml

```diff
@@ -0,0 +1,79 @@
+name: '.Platform: run pester test for convert to ARM template script'
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'utilities/tools/ConvertTo-ARMTemplate**'
+
+env:
```

Comment:
Do we need any of those `env` values?

Reply:
No. Removed them. new run: [![.Platform: run pester test for convert to ARM template script](https://github.com/Azure/ResourceModules/actions/workflows/platform.convertToArmTemplate.tests.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/platform.convertToArmTemplate.tests.yml)